api-inference-community/api_inference_community/batch.py

```diff
@@ -3,19 +3,28 @@
 
 import datasets
 import tqdm
+from api_inference_community.validation import normalize_payload
 from huggingface_hub import HfApi
 
 
-def iterate(pipe, dataset, f):
-    for i, result in enumerate(tqdm.tqdm(pipe.iter(dataset))):
-        write_result(result, f)
-
-
-def iterate_slow(pipe, dataset, f):
+def iterate_slow(pipe, dataset, f, task: str):
```

Comment:
Should this be renamed to just `iterate` now?

Reply:
Done.